validation-tool-backend/src/review/review.controller.ts

```diff
@@ -73,4 +73,26 @@ export class ReviewController {
     );
     return true;
   }
+
+  @Get('/progress')
+  @UseGuards(JwtAuthGuard)
+  async getReviewProgress(
+    @Req() req
+    ) {
+    const user: any = req.user;
+    const run_ids = user?.privileges.map((x) => x.run_id);
```

Comment:
Is this not sidestepping the role assigned to the run? e.g. if a user has a reviewer role on the run_id and not an admin role, we might not show this?

Reply:
done